repos/exawind/packages/trilinos/package.py

```diff
@@ -17,6 +17,7 @@ class Trilinos(bTrilinos, CmakeExtension):
     # the trilinos github page appended with the date of the commit.
     # this preserves the Trilinos versioning scheme and will allow for valid
     # version comparisons in spack's internals.
+    version("15.1.1", tag="trilinos-release-15-1-1")
     version("14.1.0.2023.02.28", commit="8b3e2e1db4c7e07db13225c73057230c4814706f")
```

Comment:
```suggestion
    version("13.4.1.2023.02.28", commit="8b3e2e1db4c7e07db13225c73057230c4814706f")
```
This is what I mean.